woocommerce/woocommerce
# Update shipping dimensions image in new product blocks editor

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Updates the shipping images to the new design and adds labels for each side when values are filled in.

<img width="852" alt="Screen Shot 2023-05-03 at 2 10 17 PM" src="https://user-images.githubusercontent.com/10561050/236051582-e2b87f06-09fc-4682-9899-c12a6400c102.png">


Closes #38043  .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Navigate to Tools -> WCA Test Helper -> Features and enable the new product blocks editing experience
2. Navigate to Products -> Add new
3. Click on the "Shipping" tab
4. Click on the various shipping fields, making sure the corresponding SVG lines/sides are highlighted when focusing an input
5. Fill in the input with some values
6. Check that these values show next to the respective areas in the SVG
